.eslintrc.json

```diff
@@ -48,45 +46,36 @@
             { "selector": "property", "format": null }
         ],
 
-        "@typescript-eslint/consistent-type-definitions": ["error", "interface"],
-        "@typescript-eslint/consistent-type-assertions": ["error", { "assertionStyle": "as" }],
-
-        "max-statements-per-line": ["error", { "max": 1 }],
-
-        "no-duplicate-imports": "off",
-        "@typescript-eslint/no-duplicate-imports": "error",
-
-        "@typescript-eslint/no-inferrable-types": "error",
-        "@typescript-eslint/no-misused-new": "error",
-        "@typescript-eslint/no-this-alias": "error",
-
-        "no-unused-expressions": "off",
-        "@typescript-eslint/no-unused-expressions": ["error", { "allowTernary": true }],
-
-        "@typescript-eslint/prefer-for-of": "error",
-        "@typescript-eslint/prefer-function-type": "error",
-        "@typescript-eslint/prefer-namespace-keyword": "error",
-        "@typescript-eslint/prefer-as-const": "error",
-
-        "quotes": "off",
+        "@typescript-eslint/brace-style": ["error", "stroustrup", { "allowSingleLine": true }],
+        "@typescript-eslint/no-extra-semi": "error",
         "@typescript-eslint/quotes": ["error", "double", { "avoidEscape": true, "allowTemplateLiterals": true }],
-
-        "semi": "off",
         "@typescript-eslint/semi": "error",
-        "@typescript-eslint/no-extra-semi": "error",
-
-        "space-before-function-paren": "off",
         "@typescript-eslint/space-before-function-paren": ["error", {
             "asyncArrow": "always",
             "anonymous": "always",
             "named": "never"
         }],
 
-        "@typescript-eslint/triple-slash-reference": "error",
-        "@typescript-eslint/type-annotation-spacing": "error",
-        "@typescript-eslint/unified-signatures": "error",
-
-        "@typescript-eslint/no-extra-non-null-assertion": "error",
+        // Todo: do we want these?
+        "prefer-rest-params": "off",
+        "prefer-spread": "off",
+        "@typescript-eslint/ban-types": "off",
+        "@typescript-eslint/no-empty-function": "off",
+        "@typescript-eslint/no-empty-interface": "off",
+        "@typescript-eslint/no-unused-vars": "off",
```

Comment:
Is this off because the tsconfig has the equivalent Typescript setting on? If not, what kind and how many unused variables do we have?

Reply:
Yes, it's this; honestly I would _much_ prefer if we didn't use TS for this at all and instead made it a lint. So annoying to have to ensure my variables are all used just to run tests or something. At least debug skips typecheck.

Reply:
Only 18 failures at the moment. Not bad!

However, https://github.com/typescript-eslint/typescript-eslint/issues/5017 -> https://github.com/gajus/eslint-plugin-jsdoc/issues/858 is impacting the codebase here. Probably best to set up the workarounds in a separate PR IMO.